### ISSUE TITLE
Uncaught TypeError: Cannot read property '0' of null

### ISSUE THREAD

**Issue by rskuipers** [RobQuistNL/chrome-github-jira#3]
```
chrome-extension://faenbbkpfnklhncjianlfllkfekgghih/src/content.js:82 Uncaught TypeError: Cannot read property '0' of null
```

This error triggers multiple times on opening a PR on an open source project (so without ticket number in title).

**Comment by RobQuistNL** [comment]
:+1: Thanks for the feedback :)

**Comment by RobQuistNL** [comment]
Will be fixed in 1.0.4
